packages/aws-cdk-lib/aws-codedeploy/lib/base-deployment-config.ts

```diff
@@ -153,6 +209,13 @@ export abstract class BaseDeploymentConfig extends Resource implements IBaseDepl
 
     this.node.addValidation({ validate: () => validateName('Deployment config', this.physicalName) });
   }
+
+  private validateDurationLessThanOneSecond(duration: Duration, name: string) {
```

Comment:
Could we rename this to `validateDurationGreaterThanOneSecond` or `validateMinimumDuration`?

having the method name start with `validate` feels like it makes more sense to validate that the input meets some requirement, not the opposite case where you're validating against the invalid values. What do you think?

Reply:
I've updated func name!